package.json

```diff
@@ -25,9 +25,12 @@
     "db:update-client": "npx prisma generate",
     "spec:generate": "cross-env NODE_ENV=development ts-node scripts/generateOpenApi.ts && npm run spec:validate",
     "spec:validate": "cross-env NODE_ENV=development ts-node scripts/validateOpenApi.ts",
-    "prepublishOnly": "npm run build"
+    "prepublishOnly": "npm run build",
+    "prepush": "npm run test:coverage && npm run audit",
```

Comment:
we have CI for ensuring code coverage, we don't need a prepush hook for this.
we shouldn't be running audit locally either, we can have GitHub workflow instead